File: superagentweb/src/main/java/com/siyee/superagentweb/abs/IndicatorController.java
package com.siyee.superagentweb.abs;

import android.webkit.WebView;

public interface IndicatorController {

    void progress(WebView v, int newProgress);

    BaseIndicatorSpec offerIndicator();

    void showIndicator();

    void setProgress(int newProgress);

    void finish();

}

File: superagentweb-core/src/main/java/com/siyee/superagentweb/IndicatorController.kt
package com.siyee.superagentweb

import android.webkit.WebView




interface IndicatorController {

    fun progress(v: WebView?, newProgress: Int)

    fun offerIndicator(): BaseIndicatorSpec

    fun showIndicator()

    fun setProgress(newProgress: Int)

    fun finish()

}